egs/puebla_nahuatl/st1/local/normalize_punctuation.pl

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/perl
```

Comment:
This file is not necessary.

Reply:
Thanks! Please pardon me for my mis-check.